pkg/k8s-client/client.go

```diff
@@ -30,27 +30,24 @@ type client struct {
 
 // NewK8sClient returns a kubernetes client
 func NewK8sClient() (Client, error) {
-	glog.V(3).Info("NewK8sClient():")
+	log.Info().Msg("NewK8sClient():")
 	// Get a config to talk to the api server
-	glog.Info("Setting up kubernetes client")
+	log.Debug().Msg("Setting up kubernetes client")
```

Comment:
what is the added value of this log message ?

Reply:
removed the previous log (NewClient)